src/constants.ts

```diff
@@ -89,9 +94,33 @@ export const kChildCollectionAttributes = [
     title: "Season",
     type: "categorical",
     hasToken: true
+  },
```

Comment:
If project team wants it, we can make these formulas rather than calculating them in the plugin